setup.cfg

```diff
@@ -68,6 +68,7 @@ exclude =
 
 # Additional dependencies for specific services
 all =
+    challengeutils~=4.3
     synapseclient~=2.7
```

Comment:
Can we bump this to the latest version?

Reply:
This is creating a dependency conflict that I'm fiddling with.

Reply:
OK, don't spend too much time on it if its being a pain. we can always create a separate ticket

Reply:
Yeah let's make a separate ticket for this for the sake of moving this along.